server/presence.go

```diff
@@ -0,0 +1,50 @@
+package main
```

Comment:
Good feedback -- your intuition is right on the boundary between `msteams` and `main`. For the most part, we're trying to keep business logic out of `msteams` and otherwise flatten most everything else into `main`.

We went through a cleanup in the past few weeks eliminating layers of interfaces and packages in favour of something simpler to maintain, but might split out in the future if this project has legs.